draft-ietf-quic-transport.md

```diff
@@ -124,10 +124,56 @@ provides:
 
 * Authenticated and encrypted header and payload
 
-QUIC uses UDP as a substrate to avoid requiring changes to legacy client
-operating systems and middleboxes.  QUIC authenticates all of its headers and
-encrypts most of the data it exchanges, including its signaling, to avoid
-incurring a dependency on middleboxes.
+QUIC establishes a connection, which is a stateful interaction between a client
+and server. The primary purpose of a connection is to support the exchange of
+data by an application protocol.
+
+Streams ({{streams}}) are the primary means by which an application protocol
+exchanges information.  Streams are ordered sequences of bytes.  Two types of
+stream can be created: bidirectional streams allow both endpoints to send data;
+unidirectional streams allow a single endpoint to send.  Endpoints can create
+streams without prior negotiation.  A credit-based scheme is used to limit
+stream creation and to bound the amount of data that can be sent.
+
+The QUIC handshake combines negotiation of cryptographic and transport
+parameters.  The handshake is structured to permit the exchange of application
+data as soon as possible.  This includes an option for clients to send data
+immediately (0-RTT) as possible, which might require prior communication to
+enable.
+
+QUIC connections are not strictly bound to a single network path.  Connection
+migration uses connection identifiers to allow connections to transfer to a new
+network path.
+
+Frames are used in QUIC to communicate between endpoints.  Multiple frames are
+assembled into packets.  QUIC authenticates all packets and encrypts as much as
+is practical to avoid incurring a dependency on middleboxes.  QUIC packets are
```

Comment:
Isn't the dependency the other way around; we're trying to avoid middleboxes becoming dependent on any visible features of QUIC.